### PR TITLE
Profile ahmed jamil

### LINKED ISSUE #5292: [Community] Member Profile: Ahmed Jamil
**Current Behavior**

@AjaxAueleke has been a consistent contributor and community member.

**Desired Situation**
Let's recognize @AjaxAueleke as a contributor and community member by creating a profile on https://layer5.io/community/members.

- GitHub: @AjaxAueleke
- Twitter: 
- LinkedIn: [/in/muhammad-ahmed-97578b1ab/](https://www.linkedin.com/in/muhammad-ahmed-97578b1ab/)
- Layer5 Cloud: 
- Link to profile picture:

A detailed explanation on how to set up a community member profile can be found in the [CONTRIBUTING.md](https://github.com/layer5io/layer5/blob/master/CONTRIBUTING.md)

---

**Contributor Resources**

The layer5.io website uses Gatsby, React, and GitHub Pages. Site content is found under the [`master` branch](https://github.com/layer5io/layer5/tree/master).
- See [contributing instructions](https://github.com/layer5io/layer5/blob/master/CONTRIBUTING.md)
- See Layer5 site designs in this [Figma project](https://www.figma.com/file/5ZwEkSJwUPitURD59YHMEN/Layer5-Designs). Join the [Layer5 Community](https://slack.layer5.io) for access.

### PR DESCRIPTION
**Pull request by srujan0404**
**Description**
Kindly check and review my PR, I have added Ahmed jamil in the members list under community

This PR fixes #5292

**Notes for Reviewers**

**[Signed commits](https://github.com/layer5io/layer5/blob/master/CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [ ] Yes, I signed my commits.

### CODE REVIEW

**Comment by l5io** [comment]
🚀 Preview for commit 474195ef968e62bf5f3d6218eb99a3f130d829cb at: https://65a95b9e7dc1680319a287cc--layer5.netlify.app

File: src/collections/members/ahmed-jamil/index.mdx [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+position: Contributor
+image_path: ./ahmed-jamil.webp
+github: AjaxAueleke
+twitter: mo_ahmed_jamil
+linkedin: muhammad-ahmed-97578b1ab
+layer5: 93770230-940d-43eb-bd97-0a40bdd4ec5d
+location: Karachi, Pakistan
+bio: Hi! I'm Ahmed Jamil, an enthusiastic software engineer and undergraduate student diving deep into the world of open source and cloud-native technologies.
+badges:
+- community
```
Comment (iArchitSharma) [inline src/collections/members/ahmed-jamil/index.mdx:12]: @srujan0404 remove the community badge

**Review by iArchitSharma — CHANGES_REQUESTED** [review]
LGTM if above changes are done!

**Comment by srujan0404** [comment]
@iArchitSharma Done with changes

**Comment by l5io** [comment]
🚀 Preview for commit 78305526d5ce7b5ef246745609c92b18b90d1b21 at: https://65aa790f7cad7b08caa2dac7--layer5.netlify.app

**Review by samyakshah3008 — APPROVED** [review]
Looks good ✅

**Comment by l5io** [comment]
🚀 Preview for commit 3da67c38ac7273395cc1e514e34b6bc6e4bb6119 at: https://65afa37ae900d00393c3c9db--layer5.netlify.app

**Comment by sudhanshutech** [comment]
Thanks @srujan0404

**Pull request merged**